src/js/profile/subindicator_filter.js

```diff
@@ -143,27 +162,18 @@ export class SubindicatorFilter extends Observable {
         });
 
         let chartData = [];
```

Comment:
this can be removed since it is not used anymore.

Reply:
What isn't being used anymore?

Reply:
`chartData` you only used it to push data onto and return empty.